bin/mark_outliers_in_json

```diff
@@ -21,11 +21,13 @@
 """
 
 import argparse
-import bz2
-import json
-import numpy
 import os
 import os.path
+import sys
+
+sys.path.insert(0, '../warmup')
```

Comment:
Python does have relative import syntax, it would be `from ..warmup,whatever import name` but this won't work in our case because `..` is not a package (although `warmup` is).